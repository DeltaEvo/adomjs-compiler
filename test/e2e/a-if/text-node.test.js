const compileAndCreate = require('../compileAndCreate');

const element = compileAndCreate(`<template a-if="condition">Hello</template>`);

it('render', () => {
	element.setState({
		condition: true
	});

	expect(element.innerHTML).toMatchSnapshot();
});

it('remove elements', () => {
	element.setState({
		condition: false
	});

	expect(element.innerHTML).toMatchSnapshot();
});
