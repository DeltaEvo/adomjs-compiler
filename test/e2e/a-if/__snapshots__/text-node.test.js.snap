// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`remove elements 1`] = `""`;

exports[`render 1`] = `""`;
